### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by kbarkhqs**

### CODE REVIEW

File: documentation/src/container_types/operators_hamiltonians_and_systems.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ Operators and systems act on a state space using HashMaps (Dictionaries) of oper
 
 For spins, the operators and systems represent
 \\[ 
-\hat{O} = \sum_{j=0}^N \alpha_j \prod_{k} \sigma^{k}_j \\\\
-    \sigma^{k}_j \in \\{ X, Y, Z, I \\}
+\hat{O} = \sum_{j} \alpha_j \prod_{k=0}^N \sigma_{j, k} \\\\
+    \sigma_{j, k} \in \\{ X_k, Y_k, Z_k, I \\}
 \\]
-where the \\(\sigma^{k}_j\\) are `SinglePauliOperators`.
+where the \\(\sigma_{j, k}\\) are `SinglePauliOperators`.
```
Comment (rreiner-hqs) [inline documentation/src/container_types/operators_hamiltonians_and_systems.md:10]: Any particular reason for putting both indices down on the sigmas? 
Also, I would add the `k` subscript also on the identity: `I_k`

File: documentation/src/physical_types/mixed_systems.md [hunk 2]
```diff
@@ -109,16 +109,16 @@ let mdp = MixedDecoherenceProduct::new([dp], [bp], [fp]).unwrap();
 Complex objects are constructed from operator products are `MixedOperators` and `MixedHamiltonians`
 (for more information, [see also](../container_types/operators_hamiltonians_and_systems.md)).
 
-These `MixedOperators` and `MixedHamiltonians` represent operators or hamiltonians such as:
-\\[ \hat{H} = \sum_{j=0}^N \alpha_j \prod_{k} \sigma_j^{k}  \prod_{l, m} c_{b, l, j}^{\dagger} c_{b, m, j} \prod_{r, s} c_{f, r, j}^{\dagger} c_{f, s, j} \\]
+These `MixedOperators` and `MixedHamiltonians` represent operators or Hamiltonians such as:
+\\[ \hat{H} = \sum_j \alpha_j \prod_k \sigma_{j, k}^{a} \prod_{l, m} c_{b, l, j}^{\dagger} c_{b, m, j} \prod_{r, s} c_{f, r, j}^{\dagger} c_{f, s, j} \\]
```
Comment (rreiner-hqs) [inline documentation/src/physical_types/mixed_systems.md:113]: What is the new exponent/superscript `a` here?

File: documentation/src/physical_types/spins.md [hunk 3]
```diff
@@ -123,17 +123,17 @@ let dp_equivalent = DecoherenceProduct::new()
 A good example how complex objects are constructed from operator products are `SpinOperators` and `SpinHamiltonians`
 (for more information, [see also](../container_types/operators_hamiltonians_and_systems.md)).
 
-These `SpinOperators` and `SpinHamiltonians` represent operators or hamiltonians such as:
+These `SpinOperators` and `SpinHamiltonians` represent operators or Hamiltonians such as:
 \\[
-\hat{O} = \sum_{j=0}^N \alpha_j \prod_{k} \sigma^{k}_j \\\\
-    \sigma^{k} \in \\{ X, Y, Z, I \\}
+\hat{O} = \sum_{j} \alpha_j \prod_{k=0}^N \sigma_{j, k} \\\\
+    \sigma_{j, k} \in \\{ X_k, Y_k, Z_k, I \\}
```
Comment (rreiner-hqs) [inline documentation/src/physical_types/spins.md:129]: Same thing about `I_k` here

File: documentation/src/container_types/operators_hamiltonians_and_systems.md (hunk 1 above)
Comment (kbarkhqs) [reply documentation/src/container_types/operators_hamiltonians_and_systems.md:10]: Yes, the equation itself has been modified with Nico and we felt this was best

File: documentation/src/physical_types/mixed_systems.md (hunk 2 above)
Comment (kbarkhqs) [reply documentation/src/physical_types/mixed_systems.md]: A mistake, thank you! Removed

**Review by rreiner-hqs — APPROVED** [review]

**Pull request merged**